backend/src/resource/notificacao-resource.js

```diff
@@ -25,13 +25,17 @@ const buscarPessoaDadosBasicos = async (nome, nomeDaMae) => models.Pessoa.findAl
 });
 
 const consolidarSuspeito = async (suspeito) => {
-  const {
+  var {
     pessoaId, bairroId, MunicipioId, nome, nomeDaMae,
+    sexo, gestante, corRaca
   } = suspeito;
-  const suspeitoPrototipo = { bairroId, MunicipioId };
+  var suspeitoPrototipo = { bairroId, MunicipioId };
```

Comment:
Para limitar o contexto, seria melhor utilizar let
```suggestion
  let suspeitoPrototipo = { bairroId, MunicipioId };
```

Reply:
Alterado.